models/netsuite2/netsuite2__balance_sheet.sql

```diff
@@ -50,13 +51,17 @@ balance_sheet as (
             and {{ dbt.date_trunc('year', 'reporting_accounting_periods.starting_at') }} = {{ dbt.date_trunc('year', 'transaction_accounting_periods.starting_at') }} 
             and reporting_accounting_periods.fiscal_calendar_id = transaction_accounting_periods.fiscal_calendar_id) then 'Net Income'
       when not accounts.is_balancesheet then 'Retained Earnings'
+      when lower(accounts.special_account_type_id) = 'retearnings' then 'Retained Earnings'
+      when lower(accounts.special_account_type_id) IN ('cta-e', 'cumultransadj') then 'Cumulative Translation Adjustment'
       else accounts.type_name
         end as account_type_name,
     case
       when (not accounts.is_balancesheet 
             and {{ dbt.date_trunc('year', 'reporting_accounting_periods.starting_at') }} = {{ dbt.date_trunc('year', 'transaction_accounting_periods.starting_at') }} 
             and reporting_accounting_periods.fiscal_calendar_id = transaction_accounting_periods.fiscal_calendar_id) then 'net_income'
       when not accounts.is_balancesheet then 'retained_earnings'
+      when lower(accounts.special_account_type_id) = 'retearnings' then 'retained_earnings'
+      when lower(accounts.special_account_type_id) IN ('cta-e', 'cumultransadj') then 'cumulative_translation_adjustment'
```

Comment:
Same question about if this could conflict with the union logic for cta?

Reply:
No conflict and same logic noted above.